components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -181,8 +215,9 @@ public IdentityProvider getResidentIdP(String tenantDomain)
             samlECPUrl = IdentityUtil.getServerURL(IdentityConstants.ServerConfig.SAMLSSO, true, true);
         }
 
-        samlAuthnRequestsSigningEnabled = IdentityUtil.getServerURL(IdentityConstants.ServerConfig.
-                SAML_METADATA_AUTHN_REQUESTS_SIGNING_ENABLED, true, true, true, false);
+        samlAuthnRequestsSigningEnabled = IdentityUtil
+                .getServerURL(IdentityConstants.ServerConfig.SAML_METADATA_AUTHN_REQUESTS_SIGNING_ENABLED, true, true) +
+                IdPManagementUtil.getTenantParameter();
```

Comment:
A wrong config ? Is there a use of this really

Reply:
I changed this to its original implementation since I had changed it to use the modified getServerURL method. Shall we remove this since it is not used?

The property is used here [1]. But it is not set to the FederatedAuthnConfig from the IdentityProviderManager impl.

 [1] - []()https://github.com/wso2-extensions/identity-metadata-saml2/blob/41cc848d17f32d5fc377557de91f6efe7473ecb2/components/org.wso2.carbon.identity.idp.metadata.saml2/src/main/java/org/wso2/carbon/identity/idp/metadata/saml2/builder/IDPMetadataBuilder.java#L57